src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisController.java

```diff
@@ -1,8 +1,6 @@
 package ca.corefacility.bioinformatics.irida.ria.web.analysis;
 
 import java.security.Principal;
-import java.util.Date;
-import java.util.Optional;
 
 import org.slf4j.Logger;
```

Comment:
I am seeing warnings in this file that several of the auto wired components are not being used (emailController, analysisAudit, analysisTypeService), can you clean this up?

Reply:
Removed in 8d63f91593b46a445bb918bea869f802e346b15f